glfs.c

```diff
@@ -196,6 +194,33 @@ static int gluster_cache_add(gluster_server *dst, glfs_t *fs, char* cfgstring)
 	return -1;
 }
 
+static bool gluster_cache_update_logdir(const char *logdir)
```

Comment:
Should `logdir` not be used somewhere in this function? Now it is only printed when there is an error.

Reply:
Yeah, right and will fix it.
This is from the old version and forget to remove it.
Thanks.